gandalf.js

```diff
@@ -9,16 +9,13 @@
  * Copyright 2016 Henrik Ingo (@henrikingo)
```

Comment:
SCRIPT-Consider renaming this variable to something more descriptive for better readability.